packages/client/components/Dashboard/LeftDashNavItem.tsx

```diff
@@ -85,7 +77,7 @@ const LeftDashNavItem = (props: Props) => {
         {
           {
             forum: <Forum />,
-            timeline: <Timeline />,
+            timeline: <History />,
```

Comment:
You need to change the mapping here to `history: <History />,` if you want to pass `history` in `DashSidebar`.

Reply:
@Dschoordsch Ah, got it! Thank you 👍